Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/fileHandlerFactory/TextFileHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.cdp.volha_ivanova.task1.fileHandlerFactory;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class TextFileHandler extends FileHandler {
+
+	private PrintWriter out = null;
+
+	public void writeToFile() throws IOException {
```

Comment:
I cant see big difference between your TextFileHandler and XMLFileHandler. Actually you can have only one that take fileName as parameter. But now keep as you have to try factory pattern.